learning-development/learning-support.qmd

```diff
@@ -48,23 +53,76 @@ In this workshop, we cover:
 
 Contact [statistics.development@education.gov.uk](mailto:statistics.development@education.gov.uk) to register interest in workshops happening at your site or to request new topics!
 
-## Partnership programme
+---
 
- *	Dedicated resource to support development work, either building skills or working on analytical or statistical outputs.
- *	A short-term project with an agreed set of goals and timescale decided as part of an initiation meeting.
- * Contact statistics.development@education.gov.uk if you're interested in finding out more about the support we can give.
+## Partnership programmes
 
-## Specific resources
+The Statistics Development Team offers bespoke partnership programmes for analyst teams across the department, designed as dedicated resource to support development work, either building skills or filling resource gaps to allow improvements in analytical or statistical outputs.
 
-We also have specific learning resources and materials for [SQL](../learning-development/sql.html), [R](../learning-development/r.html), and [Git](../learning-development/git.html)
+Partnerships are designed as short-term projects, agreed in conjunction with the team and their Grade 6. The maximum length for the majority of partnership programmes will be three months, and the project must be clearly defined at the outset.
+
+The aim of partnership programmes is always to upskill the team throughout the project. For this reason, we require any teams taking part to commit time to upskilling during the partnership and being willing to independently continue the work after the partnership is complete, if appropriate. 
+
+---
+
+### What could a partnership programme involve?
+
+---
+
+- Flexible skilled resource for your team to support with R, Git, SQL or Databricks
+- Help with dedicated upskilling 
+- Knowledge sharing, including around best practice and accessibility, and peer review
+- Support with RAP implementation to allow your team to meet the RAP baseline or best practice criteria
+
+For example, this might be:
 
+- Automating processes such as data manipulation or QA
+- Creation of R Shiny dashboards
+- Reviewing existing analytical code
+- Support with moving to Azure DevOps or GitHub
 
 ---
 
-# General resources
+### What types of partnership are there?
 
 ---
 
+We have four levels of support available:
+
+- One-off bespoke support via a team meeting or a workshop, or one-off peer review 
+- **Tier 1**: occasional check-in meetings to provide best practice guidance 
+- **Tier 2**: hands on support for teams with skills or resource gaps 
+- **Tier 3**: designated resource working within your team to write code / complete a project for you, designed for complex projects
+
+Teams participating in tiers 1-3 will be assigned a named person from the Statistics Development Team who will support you in completing the project. 
+
+---
+
+### What is the process to get started with a partnership?
```

Comment:
Could be `How to get started` - I think you've written these headings very FAQ style, and that tends to make them extra long compared to what they could be (very minor thing though!)

Reply:
Have changed it but not sure the new version is any shorter 😆

Reply:
Give we're in the heading / section called partnership programme I think you can change it from:

`How to get started with a partnership programme`

to

`How to get started`

Reply:
Done! it sounded weird the first time I changed it to that but upon reflection it is fine